src/openfermion/utils/_channel_state.py

```diff
@@ -0,0 +1,184 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Module to manipulate basic models of quantum channels"""
+
+from functools import reduce
+from itertools import chain
+from numpy import array, dot, eye, kron, log2, sqrt
+
+
+def _verify_channel_inputs(density_matrix, probability, target_qubit):
+    """Verifies input parameters for channels
+
+    Args:
+        density_matrix(numpy.ndarray): Density matrix of the system
+        probability(float): Probability error is applied p \in [0, 1]
+        target_qubit(int): target for the channel error.
+
+    Returns:
+        new_density_matrix(numpu.ndarray): Density matrix with the channel
+            applied.
+    """
+    n_qubits = int(log2(density_matrix.shape[0]))
+
+    if (density_matrix.shape[0] != density_matrix.shape[1] or
+            len(density_matrix.shape) != 2):
+        raise ValueError("Error in input of density matrix to channel.")
+    if (probability < 0) or (probability > 1):
+        raise ValueError("Channel probability must be between 0 and 1.")
+    if (target_qubit < 0) or (target_qubit >= n_qubits):
+        raise ValueError("Target qubits must be within number of qubits.")
+
+
+def _lift_operator(operator, n_qubits, target_qubit):
+    """Lift a single qubit operator into the n_qubit space by kron product
+
+    Args:
+        operator(ndarray): Single qubit operator to lift into full space
+        n_qubits(int): Number of total qubits in the space
+        target_qubit(int): Qubit to act on
+
+    Return:
+        new_operator(Sparse Operator): Operator representing the embedding in
+            the full space.
+    """
+    new_operator = reduce(kron,
```

Comment:
Could you maybe respace this? If you put kron on a new line you might be able to fit the `eye(2) for i in range(...)` on a single line - I think that would make this easier to read